fw102/fw102.ibek.support.yaml

```diff
@@ -0,0 +1,62 @@
+# yaml-language-server: $schema=../schemas/ibek.support.schema.json
+
+module: fw102
+
+entity_models:
+
+  - name: fw102
+    description: |-
+      Create an fw102 device
```

Comment:
In general what should this description be for? Should it explain what the device actually is, or is that not interesting in this context?